### ISSUE TITLE
Options in settings no longer working

### ISSUE THREAD

**Issue by MED1975** [Brandawg93/homebridge-nest-cam#67]
The optional settings are no longer reflected properly in HomeKit. I have the following in my config.json 

                "motionDetection": true,
                "streamingSwitch": false,
                "doorbellAlerts": true,
                "doorbellSwitch": false,
                "disableAudio": false

However, the doorbell switch is still showing along with the streaming switch.

This has been happing for the last few versions released. If I enable the streaming switch the config.json changes to true but the accessory remains.

Thanks!

**Comment by Brandawg93** [comment]
I've tested this myself and have asked a few other users who have confirmed that theirs is working. Make sure these are wrapped in an object called `options`.
```
            "options": {
                "motionDetection": true,
                "streamingSwitch": true,
                "doorbellAlerts": true,
                "doorbellSwitch": true,
                "disableAudio": false
            },
```
If that doesn't work, you can try deleting your `accessories` and `persist` folders.

**Comment by MED1975** [comment]
Hi. I would rather not remove those folders as I will have a lot of work to add everything back with scenes and automations. 

My config.json is formatted correctly and I used the settings option in config ux to adjust the plugin settings. 

This behaviour has been there since a recent update where you mentioned a change to the settings file? Maybe two releases ago. 

I have also removed and added back the plugin and it hasn’t helped. Is there any other info I can give you?

So to recap the options in settings are correct and they match the config file But this change doesn’t seem to make any difference when I restart the server.

**Comment by Brandawg93** [comment]
I've published a new test version that should fix this. You can install it via `npm -g i homebridge-nest-cam@test`. Let me know if this fixes it for you.

**Comment by Brandawg93** [comment]
I renamed the `options` key in config by mistake. Try the new release v2.3.4 and let me know if it works for you.

**Comment by Brandawg93** [comment]
Others have confirmed this is working. If not, just reopen this ticket and I'll take a look.

**Comment by MED1975** [comment]
That’s great now. All super duper. Thanks for the quick fix.
